electricpy/__init__.py

```diff
@@ -1636,6 +1636,91 @@ def curdiv(Ri, Rset, Vin=None, Iin=None, Vout=False, combine=True):
     else:
         return (Ii)
 
+#Define Inductive voltage divider
+def inductive_voltdiv(Vin=None, Vout=None, L1=None, L2=None, find=''):
+    r"""
+    Inductive voltage divider.
+
+    Inductive voltage divider Inductive voltage dividers are made out of two inductors. 
+    One of the inductors is connected from the input to the output and the other one is connected from the output to ground. 
+    You can also use other components like resistors and inductors.
+
+    .. math:: V_{out} = \frac{V_{in}*L1}{L1+L2}
+
+    .. image:: /static/inductive-voltage-divider-circuit.png
+
+    Parameters
+    ----------
+    Vin:    float, optional 
+            The input voltage for the system, default=None
+
+    Vout:   float, optional
+            The output voltage for the system, default=None
+
+    L1:     float,optional
+            Value of the inductor above the output voltage, default=None
+
+    L2:     float,optional
+            Value of the inductor below the output voltage
+
+    find:   str, optional
+            Control argument to specify which value
+            should be returned.
+    
+    Returns
+    -------
+    Vin:    float, optional 
+            The input voltage for the system, default=None
+
+    Vout:   float, optional
+            The output voltage for the system, default=None
+
+    L1:     float,optional
+            Value of the inductor above the output voltage, default=None
+
+    L2:     float,optional
+            Value of the inductor below the output voltage
+    """
+    if Vin!=None and L1!=None and L2!=None:
+        Vout = (Vin*L1)/(L1+L2)
+    elif Vout!=None and L1!=None and L2!=None:
+        Vin = (Vout)*(L1+L2)/(L1)
+    elif Vin!=None and Vout!=None and L2!=None:
+        L1 = L2*(Vin -Vout)/(Vout)
+    elif Vin!=None and Vout!=None and L1!=None:
+        L2 = L1*(Vout)/(Vin - Vout)
+    else:
+        raise ValueError("ERROR: Invalid Parameters or too few" +
+                        " parameters given to calculate.")
+    
+    if find == 'Vin':
```

Comment:
Consider using `find.lower()` for each of the conditional statements here. That way, any variation of the string (e.g., 'vIN') can be used.